styles/fonts.scss

```diff
@@ -59,10 +59,12 @@ $font-base-weight: 400;
 $font-base-style: normal;
 $font-base-variant: normal;
 
+
+
 @font-face {
-  font-family: 'Visby';
-  src: url('~/public/assets/fonts/VisbyCF-Regular.otf');
-  font-weight: 400;
+  font-family: 'Visby' serif;
+  src: url('../public/assets/fonts/VisbyRegular.otf');
+  font-weight: 500;
   font-style: normal;
 }
```

Comment:
Please check these Vercel docs to load custom fonts
https://nextjs.org/docs/basic-features/font-optimization
https://solutions-loading-web-fonts.vercel.app/